providers/site.rb

```diff
@@ -46,7 +46,7 @@
     configure
 
     if new_resource.application_pool
-      shell_out!("#{appcmd(node)} set app \"#{new_resource.site_name}/\" /applicationPool:\"#{new_resource.application_pool}\"", returns: [0, 42])
+      shell_out!("#{appcmd(node)} set site /site.name:\"#{new_resource.site_name}\" /[path='/'].applicationPool:\"#{new_resource.application_pool}\"", returns: [0, 42])
```

Comment:
I believe this used to be this way because this provider supported both sites and applications.  This should be resolved since there is now a provider for applications